Source/WebKit/UIProcess/WebContextSupplement.h

```diff
@@ -48,7 +48,8 @@ class WebContextSupplement {
     {
     }
 
-    WebProcessPool* processPool() const { return m_processPool.get(); }
+    WebProcessPool* processPool() { return m_processPool.get(); }
+    RefPtr<WebProcessPool> protectedProcessPool() { return processPool(); }
```

Comment:
These can probably still be const functions.

Reply:
Chris recommended that functions which return a non-const pointer to a member variable be non-const themselves since returning the pointer means the caller can modify the member variable.